Formula/devx-logs.rb

```diff
@@ -0,0 +1,11 @@
+class DevxLogs < Formula
+  desc "A small tool to deep-link to Central ELK."
+  homepage "https://github.com/guardian/devx-logs"
+  version "0.0.1"
+  url "https://github.com/guardian/devx-logs/releases/download/cli-v0.0.1/devx-logs"
+  sha256 "1da5956140a126edb045e6edaf937aa1bc188ec6126da3455ba2ae27f2fe1aff"
```

Comment:
This is the checksum of the binary at https://github.com/guardian/devx-logs/releases/tag/cli-v0.0.1, obtained via:

```console
➜ curl -L https://github.com/guardian/devx-logs/releases/download/cli-v0.0.1/checksum.txt
1da5956140a126edb045e6edaf937aa1bc188ec6126da3455ba2ae27f2fe1aff  devx-logs
```

Reply:
Naiive/Paranoid question: Is there any risk to having the checksum we are using exposed in the repo?

Reply:
> Naiive/Paranoid question: Is there any risk to having the checksum we are using exposed in the repo?

There is no risk here. This is the SHA-256 checksum of the binary file in [the release](https://github.com/guardian/devx-logs/releases/tag/cli-v0.0.1). The value is public knowledge.

IIUC we could omit this value, however it's better to include it as homebrew will perform a comparison on install, and fail if necessary. This offers some protection, should the binary be mutated at source or in transit.